src/Workspaces/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1,5 +1,7 @@
 Microsoft.CodeAnalysis.CodeStyle.NotificationOption.Severity.get -> Microsoft.CodeAnalysis.ReportDiagnostic
 Microsoft.CodeAnalysis.CodeStyle.NotificationOption.Severity.set -> void
+Microsoft.CodeAnalysis.Solution.AddDocuments(System.Collections.Generic.IEnumerable<Microsoft.CodeAnalysis.DocumentInfo> documentInfos) -> Microsoft.CodeAnalysis.Solution
```

Comment:
would it be better to force this to be a realized ImmutableArray?

Reply:
Agree - I worry about somebody using a deferrred execution yield enumerator that ends up executing on a different thread than the caller expects, etc.